lib/storage.rb

```diff
@@ -0,0 +1,26 @@
+require 'json'
+
+class Storage
```

Comment:
- [ ] It looks like there is an error while saving the games into the JSON file. When I create the game, it works fine, everything shows in the console, but once I quit the app, the data from the games are not being properly saved. I recorded a short video to show you what is happening right now:

https://user-images.githubusercontent.com/59100477/170492588-85c53701-73b9-4ac0-acbe-967ecb250917.mp4

Please take a look and fix this issue.

Reply:
Thanks. we found the issue. Thanks for noticing this.